packages/brokeneck-react/src/hooks/useFields.js

```diff
@@ -34,8 +39,17 @@ function getFormType(field) {
   return 'text'
 }
 
+const useStyles = makeStyles(() => ({
```

Comment:
if you don't use the theme, you don't need to provide a function to makeStyles, provide a plain object